docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -28,15 +14,31 @@ Tutorial Goals
 -  Push the workflow to GitHub
 -  Make a GitHub release
 
+About Galaxy
+------------
+
+Galaxy is a web-based platform for data analysis. Galaxy is open source and can be installed locally. Alternatively, many institutions offer access to managed Galaxy instances, free and otherwise. Different Galaxy instances come with their own advantages and users can make a choice based on their own needs.
+
+Using Existing Galaxy Servers
+-----------------------------
+
+Institutions across the world maintain Galaxy instances that are shared with the greater community and are a great way to learn and use Galaxy. Some of these instances are subsidized by grants and offered for free (such as the Galaxy US <https://usegalaxy.org/>`__, Galaxy EU <https://usegalaxy.eu/>`__, or Galaxy AU <https://usegalaxy.org.au/>`__) and some are available for a fee (such as Galaxy AnVIL <https://anvil.terra.bio/>`__ or Galaxy Pro <https://researcher.galaxyworks.io/>`__). A lot of great learning resources are available on the the Galaxy Training Network <https://training.galaxyproject.org/>`__, with help on specific questions available via Galaxy Help <https://help.galaxyproject.org/>`__.
```

Comment:
This section is hard to tell if the syntax is correct. Is this available on readthedocs?

Reply:
I don't think so. How can I go about doing that?

Reply:
Got it! I think. https://docs.dockstore.org/en/feature-update-launchwithgalaxy/

Reply:
Here we go: https://featuregalaxyupdates.readthedocs.io/en/feature-galaxyupdates/

Reply:
thanks, formatting looks good https://featuregalaxyupdates.readthedocs.io/en/feature-galaxyupdates/getting-started/getting-started-with-galaxy.html#using-existing-galaxy-servers